frontend/src/components/EntityList.vue

```diff
@@ -1,105 +1,106 @@
 <template>
-  <div class="row">
-    <div class="col-lg-2">
+  <BaseLayout>
+    <template v-slot:additional_breadcrumbs>
+      <li class="breadcrumb-item">{{ schemaName }}</li>
+      <li class="breadcrumb-item active">Entities</li>
+    </template>
+    <template v-slot:actions>
```

Comment:
This breadcrumb component has a little bit unstable behavior across pages; I don't know exactly which is intended and which is not, so here are my observations

Action   |  What is shown in breadcrumbs
---------|-----------------------------------
Open start page  | `home / --- / Entities`
Choose schema |  `home / Schema name / Entities`
Go to entity detail | None
Go back, open schema details | `home / Schema name / Details`
Click edit | `home / Schema name / Details`
Go back to details | `home / Schema name / Details` <br> `home / Schema name / Details`
Go back to entity list |  `home / Schema name / Details`  <br> `home / Schema name / Details` <br>`home / Schema name / Entities`
Refresh, choose schema | `home / Schema name / Entities`
Click edit | None
Go back, click create entity | None
Go back, open entity details, click edit | None
Go back to list, click create new schema | None
Go back, open schema details |  `home / Schema name / Details`
Click create new schema | `home / Schema name / Details` (and edit button leading to schema `undefined`)

By _go back_ here I mean _go back in browser history_

Reply:
That was indeed not so nice... But as I continued to work my way through the components, this unwanted behavior became visible at less places.

I will recheck, once I'm finished digging through the UI.

Reply:
I love self-eliminating bugs😄